integration-tests/src/test/java/org/zowe/apiml/integration/proxy/DownloadApiIntegrationTest.java

```diff
@@ -34,8 +35,7 @@ public static void beforeClass() {
 
     protected static String[] discoverableClientSource() {
         return new String[]{
-            "/discoverableclient/api/v1/get-file",
-            "/api/v1/discoverableclient/get-file"
+            DISCOVERABLE_GET_FILE
```

Comment:
Does it make sense to have it parametrized?

Reply:
Removed